<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window2DSettings</class>
 <widget class="QGroupBox" name="Window2DSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>109</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Window settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>x ∈</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>y ∈</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>]</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="yMax">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>,</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="yMin">
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>[</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>]</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="xMax">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>,</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="xMin">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>-5</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>[</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="showAxes">
     <property name="text">
      <string>Show axes</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="showGrid">
     <property name="text">
      <string>Show grid</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>xMin</tabstop>
  <tabstop>xMax</tabstop>
  <tabstop>yMin</tabstop>
  <tabstop>yMax</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>xMin</sender>
   <signal>editingFinished()</signal>
   <receiver>Window2DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xMax</sender>
   <signal>editingFinished()</signal>
   <receiver>Window2DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yMin</sender>
   <signal>editingFinished()</signal>
   <receiver>Window2DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yMax</sender>
   <signal>editingFinished()</signal>
   <receiver>Window2DSettings</receiver>
   <slot>onChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAxes</sender>
   <signal>toggled(bool)</signal>
   <receiver>showGrid</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAxes</sender>
   <signal>toggled(bool)</signal>
   <receiver>Window2DSettings</receiver>
   <slot>setShowAxes(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showGrid</sender>
   <signal>toggled(bool)</signal>
   <receiver>Window2DSettings</receiver>
   <slot>setShowGrid(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChanged()</slot>
  <slot>setShowAxes(bool)</slot>
  <slot>setShowGrid(bool)</slot>
 </slots>
</ui>
